hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -348,19 +350,11 @@ public static void setWALRootDir(final Configuration c, final Path root) {
   public static FileSystem getWALFileSystem(final Configuration c) throws IOException {
     Path p = getWALRootDir(c);
     FileSystem fs = p.getFileSystem(c);
-    // hadoop-core does fs caching, so need to propagate this if set
+    // hadoop-core does fs caching, so need to propogate this if set
```

Comment:
Old spelling is correct.

Reply:
Okay, I'll retain the spelling fixes from the reverted commits. Man you guys want your cake and to eat it as well!